xrpl/models/transactions/mptoken_issuance_create.py

```diff
@@ -107,10 +120,24 @@ def _get_errors(self: Self) -> Dict[str, str]:
                     _MAX_TRANSFER_FEE
                 )
 
+        if self.mptoken_metadata is not None and (
+            len(self.mptoken_metadata) == 0
+            or len(self.mptoken_metadata) > MAX_MPTOKEN_METADATA_LENGTH
+            or not HEX_REGEX.fullmatch(self.mptoken_metadata)
```

Comment:
In the first few lines of the `validate_mptoken_metadata` method, there is a validation check against a `HEX_REGEX` regular expression. Can we avoid this duplication of  validation check?